examples/state-scripts/wait-for-network/Sync_Enter_00

```diff
@@ -0,0 +1,934 @@
+
+
+
+
+
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
Did you forget to click 'raw' on this one? :-)

Reply:
Doh :D

Reply:
Fixed